frontend/src/components/collaborator/CollaboratorEdit.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <SettingsForm
```

Comment:
which functionality of the SettingsForm are you using in this component?

Reply:
Now renamed as DetailPane will be used for all kinds of details views, that are not own pages.